.eslintrc.js

```diff
@@ -4,9 +4,7 @@ module.exports = {
     jest: true
   },
   rules: {
-    'no-proto': 0,
-    strict: 0,
```

Comment:
It's recommended to use 'strict' rule with value 'error' to enforce strict mode. For example: 'strict': ['error', 'global']